server/main.go

```diff
@@ -50,7 +56,11 @@ func main() {
 	cfs := NewServer()
 
 	for _, d := range conf.Disks {
-		err = cfs.AddDisk(d.Name, d.Root)
+		name := d.Name
+		if name == "" {
+			name = strconv.FormatInt(rand.Int63(), 16)
```

Comment:
It cannot now, and is in TODO list. One possibility is to save the name back into the config file, so it could be read back. Another possibility is to record the name at cfs nameserver.